### DISCUSSION TITLE
Unity client library

### DISCUSSION THREAD

**Discussion by sabvdf** [plausible/analytics#896] [Anything Else]
@ukutaht / @metmarkosaric maybe interesting:
I just started writing a C# class for using Plausible in Unity.
It already works fine, just need to add some device data as headers and I'll call it complete.
Shall I then just publish it on the Unity Asset Store? (free and open source)
I think you might even get some new users out of that.
Unless either of you is a Unity Asset Store publisher and wants to do that themselves.

**Comment by ukutaht** [comment]
Awesome, thanks for writing the integration.

Feel free to own it. We have limited resources and we provide official support only for the Wordpress plugin and NPM module. All other integrations are developed and managed by the community.

**Comment by Garfounkel** [comment]
@sabvdf Any update? Would love to check out your C# wrapper for Plausible.
